app/components/language-logo.hbs

```diff
@@ -1,4 +1,6 @@
-{{#if (eq @variant "color")}}
+{{#if (and (eq @language.slug "gleam") (eq @variant "color"))}}
+  <GleamLogo @height={{144}} ...attributes />
```

Comment:
@Arpan-206 I mentioned this before, let's move this out to the tracks page

<img width="828" alt="Screenshot 2025-04-30 at 12 29 06" src="https://github.com/user-attachments/assets/999b7b19-0640-4fa1-bc24-dae657c52164" />